src/modules/floor/floor.schema.ts

```diff
@@ -56,7 +56,7 @@ export type UpdateFloorInput = z.infer<typeof updateFloorSchema>;
 export type ShowFloorParams = z.infer<typeof showFloorSchema>;
 export type DeleteFloorParams = z.infer<typeof deleteFloorSchema>;
 
-export const FloorSchema = showFloorResponseSchema;
+export const floorSchema = showFloorResponseSchema;
```

Comment:
you forgot to move it between `deleteFloorResponseSchema ` and `CreateFloorInput `

Reply:
Changes added